package.json

```diff
@@ -35,7 +35,7 @@
     "test-docker-hub-image": "VERSION=$(node -e \"process.stdout.write(require('./package.json').version)\") && docker image rm davidanson/markdownlint-cli2:v$VERSION davidanson/markdownlint-cli2:latest || true && docker run --rm -v $PWD:/workdir davidanson/markdownlint-cli2:v$VERSION \"*.md\" && docker run --rm -v $PWD:/workdir davidanson/markdownlint-cli2:latest \"*.md\"",
     "test-cover": "c8 --check-coverage --branches 100 --functions 100 --lines 100 --statements 100 npm test",
     "test-watch": "git ls-files | entr npm run test",
-    "update-snapshots": "ava --update-snapshots test/markdownlint-cli2-test-exec.js test/markdownlint-cli2-test-fs.js test/markdownlint-cli2-test-main.js",
+    "update-snapshots": "ava --timeout=1m --update-snapshots test/markdownlint-cli2-test-exec.js test/markdownlint-cli2-test-fs.js test/markdownlint-cli2-test-main.js",
```

Comment:
Please remove this. If timeouts become a problem (they haven't in the past outside random GitHub infrastructure issues), I would prefer to address it directly.

Reply:
I was just copying the timeout that you added to the test command last year: https://github.com/DavidAnson/markdownlint-cli2/commit/c18e7a43c0c3516ebb308116023e7e64a01dccb5#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

On my machine the tests do in fact timeout using the default 10s timeout

Reply:
I'm also fine removing it in _this_ PR and potentially opening another PR for that change in particular.  There's also a way to configure it "globally" for all ava runs via ava configuration in package.json, since I know how much you dislike repetition.

Reply:
You win, leave it here, please. :)